_episodes/04-changes.md

```diff
@@ -474,6 +474,13 @@ Date:   Thu Aug 22 09:51:46 2013 -0400
 ~~~
 {: .output}
 
+> ## Word-based diffing
+>
+> Sometimes, e.g. in the case of the LaTeX documment a line-wise
```

Comment:
There is a missing comma after "document":

"Sometimes, e.g. in the case of a LaTeX document, ..."

but I think it may even better not to mention the LaTeX example: most of our learners do not use it and this may make them think that the color diff is not useful for them.